### PR TITLE
Add content enricher metadata.

### PR DESCRIPTION
**Pull request by manuranga**
$subject.

### CODE REVIEW

File: content-enricher/content-enricher.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tagline: Add data from message
+desc: Content enricher adds data that was not sent by the original sender to message.
+helps: Ballerina enables additional lookups to to enrich the message, such as database lookups, REST API calls, ect.
```
Comment (LakshanWeerasinghe) [inline content-enricher/content-enricher.yml]:
```suggestion
helps: Ballerina enables additional lookups to to enrich the message, such as database lookups, REST API calls, etc.
```

File: content-enricher/content-enricher.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+       Spread operator (...) help to create new records out of existing records while enriching with additional data.
```
Comment (LakshanWeerasinghe) [inline content-enricher/content-enricher.yml]:
```suggestion
       Spread operator (...) helps to create new records out of existing records while enriching with additional data.
```

File: content-enricher/content-enricher.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+tagline: Add data from message
```
Comment (ushirask) [inline content-enricher/content-enricher.yml]: Should this be `Add data to message` ?

Comment (ushirask) [reply content-enricher/content-enricher.yml]: Could be better if `Add extra data to a message`

Comment (manuranga) [reply content-enricher/content-enricher.yml]: +1

File: content-enricher/content-enricher.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+desc: Content enricher adds data that was not sent by the original sender to message.
+helps: Ballerina enables additional lookups to to enrich the message, such as database lookups, REST API calls, etc.
```
Comment (prakanth97) [inline content-enricher/content-enricher.yml]:
```suggestion
helps: Ballerina enables additional lookups to enrich the message, such as database lookups, REST API calls, and etc.
```

File: content-enricher/content-enricher.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+       Spread operator (...) helps to create new records out of existing records while enriching with additional data.
```
Comment (DimuthuMadushan) [inline content-enricher/content-enricher.yml]:
```suggestion
       Spread operator (...) helps to create new records out of existing records while enriching them with additional data.
```

**Review by DimuthuMadushan — APPROVED** [review]

**Pull request merged**